Dfe.Identifiers.Api.Test/Api/IdentifiersEndpointTests.cs

```diff
@@ -18,15 +20,14 @@ public class IdentifiersEndpointTests : IClassFixture<ApiTestFixture>
     private ApiTestFixture Fixture { get; }
     private IdentifiersController Sut { get; }
```

Comment:
Sorry I didn't notice this before

Is there a reason we are calling a controller rather than the api through HTTP? It's closer to what a real user is going to do if we use a HTTP request, its not really that much different in terms of code, give me a shout if you need any assistance

Reply:
I can switch it to that. The main reasons it was done this way was just to set up the tests initially and check that the functions were working, and that we were covered by the cypress tests when running it as a proper api.

Reply:
can you change it, test is better the closer it is to the user experience

If you ever add dependencies to the controller it will break the contract, with an API that won't happen